src/test/java/com/fullteaching/e2e/no_elastest/functional/test/media/FullTeachingEndToEndRESTTests.java

```diff
@@ -20,6 +20,7 @@
 import com.fullteaching.e2e.no_elastest.common.BaseLoggedTest;
 import com.fullteaching.e2e.no_elastest.common.CourseNavigationUtilities;
 import com.fullteaching.e2e.no_elastest.common.exception.ElementNotFoundException;
+import dev.failsafe.internal.util.Assert;
```

Comment:
This is a new import, but I don't see any place where it is used

Reply:
I've removed this import and the assertTrue of the second comment, both were old imports